com.unity.multiplayer.mlapi/Editor/MLAPIProfiler.cs

```diff
@@ -41,28 +40,28 @@ class ProfilerContainer
 
             public byte[] ToBytes()
 			{
-				BitStream stream = new BitStream();
-				BitWriter writer = new BitWriter(stream);
+				NetworkBuffer buffer = new NetworkBuffer();
+				NetworkWriter writer = new NetworkWriter(buffer);
 				writer.WriteUInt16Packed((ushort)ticks.Length);
 
 				for (int i = 0; i < ticks.Length; i++)
 				{
-					ticks[i].SerializeToStream(stream);
+					ticks[i].SerializeToStream(buffer);
```

Comment:
I wonder, should we rename this function to SerializeToBuffer?

Reply:
ah, this API specifically takes in a stream :(